iothub/service/src/Jobs/JobRequest.cs

```diff
@@ -52,26 +52,24 @@ public class JobRequest
         [JsonProperty(PropertyName = "queryCondition")]
         public string QueryCondition { get; set; }
 
-
         /// <summary>
         /// ISO 8601 date time to start the job.
         /// </summary>
         [JsonProperty(PropertyName = "startTimeUtc")]
         [JsonConverter(typeof(IsoDateTimeConverter))]
-        // TODO: siport : validate if we want utc on the internal propname but not on the external
         public DateTime StartTimeUtc { get; set; }
 
         /// <summary>
         /// Max execution time in seconds (TTL duration).
         /// </summary>
         [JsonIgnore]
-        public TimeSpan MaxExecutionTime { get; set; }
+        public TimeSpan? MaxExecutionTime { get; set; }
 
         [JsonProperty(PropertyName = "maxExecutionTimeInSeconds", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
-        internal long MaxExecutionTimeInSeconds
+        internal long? MaxExecutionTimeInSeconds
         {
-            get => (long)MaxExecutionTime.TotalSeconds;
-            set => MaxExecutionTime = TimeSpan.FromSeconds(value);
+            get => MaxExecutionTime?.Seconds;
```

Comment:
Looks like we're changing the logic from this representing the TimeSpan as seconds to this representing the seconds component of the TimeSpan. Is this intentional? If yes, then a comment calling this out would be useful since generally we expect the SDK to use the TimeSpan total milliseconds value, and not just the TimeSpan.Seconds component.

Reply:
This was not intentional. I'll change this to MaxExecutionTime?.**Total**Seconds